english/main.go

```diff
@@ -13,6 +13,8 @@ type WordTokenizer struct {
 }
 
 var reAbbr = regexp.MustCompile(`((?:[\w]\.)+[\w]*\.)`)
+var reLooksLikeEllipsis = regexp.MustCompile(`(?:\.\s?){2,}\.`)
+var reEntities = regexp.MustCompile(`Yahoo!`)
```

Comment:
We are cheating a little bit here, I wonder what a more robust approach might look like.